src/main/java/org/opentripplanner/standalone/config/framework/json/NodeInfo.java

```diff
@@ -54,6 +56,31 @@ public record NodeInfo(
     }
   }
 
+  /**
+   * For some complex types like Map, we describe the child elements as part of the node-info of
+   * the parent. Hence, we need to skip the child when generating documentation. So, this factory
+   * method is used to generate a placeholder in these cases.
+   * <p>
+   * TODO DOC: A better way to do this is to remove this and add proper NodeInfo elements for
+   *           the child, but that requires a bit of refactoring.
+   */
+  static NodeInfo ofSkipChild(String name) {
+    return new NodeInfo(
+      name,
+      "No doc, parent contains doc.",
+      null,
+      OBJECT,
+      null,
+      null,
+      OtpVersion.NA,
+      null,
+      null,
+      false,
+      true,
+      null
+    );
```

Comment:
We could use the NodeInfoBuilder here? This long list of initialization parameters is a bit ugly. However, this can be changed in some future pull request.